docs/diagrams/AddSequenceDiagram.puml

```diff
@@ -15,10 +16,10 @@ participant ":Model" as Model MODEL_COLOR
 participant "applicant:Applicant" as Applicant MODEL_COLOR
 end box
 
-[-> LogicManager : execute(add n/Bob p/99999999 \n e/bob@example.com j/Data Analyst \n r/Interview s/Pandas  s/Python s/Java)
+[-> LogicManager : execute(add n/Bob \n p/99999999 \n e/bob@example.com \n j/Data Analyst r/Interview \n s/Java s/Python)
```

Comment:
Not sure if the diagram is too small? It looks alright for now. But if you would like to enlarge it, could consider cutting down fields?

```suggestion
[-> LogicManager : execute(add n/Bob \n p/999 \n e/bob@mail.com \n j/Analyst r/Interview \n s/Python)
```